src/app/CookieSerializer.js

```diff
@@ -3,12 +3,12 @@ import Cookies from 'cookies-js';
 
 export default {
 
-  serialize(token: ?t.String, { cookieSetter = null, key = 'AUTH_TOKEN' } = {})/*: t.Nil*/ {
+  serialize(token: ?t.String, { cookieSetter = null, key = 'AUTH_TOKEN' } = {})/* : t.Nil*/ {
     (cookieSetter || Cookies.set)(key, JSON.stringify(token));
   },
 
-  deserialize({ cookieGetter = null, key = 'AUTH_TOKEN' } = {})/*: ?t.String*/ {
+  deserialize({ cookieGetter = null, key = 'AUTH_TOKEN' } = {})/* : ?t.String*/ {
```

Comment:
Actually this is missing also in gdsm https://github.com/buildo/gdsm/blob/master/web/src/app/index.html